.gitignore

```diff
@@ -1,6 +1,7 @@
 .history/
 .dep
 *.o
+*.exe
```

Comment:
This should be pulled out and added to a different PR.  

Alternatively, you can use the global gitignore to not track exe's.

Reply:
Thank you @drashna. I have removed this change in an updated commit. You are correct; it should absolutely be in its own PR.

Reply:
(just saw your guilty spark pfp and i love it)